server/bundler/loader.js

```diff
@@ -2,18 +2,17 @@ var config = require( 'config' ),
 	utils = require( './utils' );
 
 function getSectionsModule( sections ) {
-	var dependencies = '',
+	var dependencies = "var controller = require( 'controller' );\n",
 		loadSection = '',
 		sectionLoaders = '';
 
 	if ( config.isEnabled( 'code-splitting' ) ) {
-		dependencies = [
+		dependencies += [
 			"var page = require( 'page' ),",
 			"\tlayoutFocus = require( 'lib/layout-focus' ),",
 			"\tReact = require( 'react' ),",
 			"\tLoadingError = require( 'layout/error' ),",
 			"\tclasses = require( 'component-classes' ),",
```

Comment:
Getting a `SyntaxError` when running Calpyso with this branch. You need a semicolon here after `require( 'component-classes )` (instead of that now-dangling comma).